### PR TITLE
Fixes #591 Badgeyay: Works for now

### LINKED ISSUE #591: Images for background templates not rendering
**I'm submitting a ...**
- [X] bug report
- [ ] feature request

**Current behavior:**
image

**Expected behavior:**
Rendering should happen properly

### PR DESCRIPTION
**Pull request by gabru-md**
Co-authored-by: Manish Devgan <manish.nsit8@gmail.com>
Co-authored-by: Parth Shandilya <parth1989shandilya@gmail.com>

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

It fixes badgeyay for now.

### CODE REVIEW

File: frontend/public/js/script.js [hunk 1]
```diff
@@ -18,13 +18,13 @@ $(document).ready(function () {
...
         $('.placeholder').text(i);
-        $('input[name='img-default']').val(i).trigger('change');
+        $("input[name='img-default']").val(i).trigger('change');
     });
 
     $('.font-options').click(function () {
         var i = $(this).data('item');
         $('.placeholder2').text(i);
-        $('input[name='custfont']').val(i);
+        $("input[name='custfont']").val(i);
```
[the remark this answers is not in the document]
Comment (gabru-md) [reply frontend/public/js/script.js]: Ignore this.

File: frontend/public/js/script.js [hunk 2]
```diff
@@ -18,13 +18,13 @@ $(document).ready(function () {
     $('.menu-options').click(function () {
         var i = $(this).data('item');
         $('.placeholder').text(i);
-        $('input[name='img-default']').val(i).trigger('change');
+        $("input[name='img-default']").val(i).trigger('change');
```
[the remark this answers is not in the document]
Comment (gabru-md) [reply frontend/public/js/script.js]: ignore this

**Comment by gabru-md** [comment]
Badges for Custom input are being generated now. Still looking for CSV one

**Comment by rupav** [comment]
PR link: https://pr-596-fossasia-badgeyay.surge.sh/

**Comment by rupav** [comment]
Same in chrome.
screenshot 2018-03-14 23 50 03

**Comment by vaibhavsingh97** [comment]
@rupav `surge.sh` is a static web publishing plateform. One can't run server side language

**Comment by rupav** [comment]
Oh I see, so if i wish to see changes, I need to pull his PR. 👍

**Comment by gabru-md** [comment]
@ParthS007 @mariobehling merge?
atleast get the manual functionality working till we work on finding a fix for CSV.

**Comment by vaibhavsingh97** [comment]
Hi Manish,

Manual version is working, only csv version is faliling 😅

**Comment by gabru-md** [comment]
@vaibhavsingh97 I'm asking to get this PR merged, so that atleast we can have one functionality working meanwhile we look for a fix for the CSV one? We should not keep the service inactive.
Your thoughts?
Ping @ParthS007

**Comment by ParthS007** [comment]
Merging as work in progress :+1:

**Pull request merged**